src/oaklib/implementations/gilda.py

```diff
@@ -38,3 +46,16 @@ def annotate_text(
                 curie=f"{match.term.db}:{match.term.id}",
                 label=match.term.entry_name,
             )
+
+    def annotate_file(
+        self, text_file: TextIOWrapper, configuration: TextAnnotationConfiguration = None
+    ) -> Iterator[TextAnnotation]:
+        """Annotate text in a file.
+
+        :param text_file: Text file that is iterated line-by-line.
+        :param configuration: Text annotation configuration.
+        :return: result of `annotate_test()`
```

Comment:
it's always better to say nothing than to say something with zero information content :-)

and it's better to say nothing than say something wrong :-)